src/app/components/LabeledInput/LabeledInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+import styles from './LabeledInput.module.css';
+
+type LabeledInputProps = {
+  children: ReactNode;
```

Comment:
You are using `children` as part of the placeholder. Thus, `children` is only allowed to be a string.
But it's possible to add other HTML elements as children because of the type `ReactNode`.

Anyway, I would rework the properties and split it in two:

```suggestion
  label: string;
  placeholder: string;
```

If you like to allow more complex labels, use `label: ReactNode`.